client/bio-commandline/src/test/kotlin/uk/ac/ebi/biostd/client/cli/services/SubmissionServiceTest.kt

```diff
@@ -56,35 +57,59 @@ internal class SubmissionServiceTest {
         verify(exactly = 1) { bioWebClient.deleteSubmission(requestDelete.accNo) }
     }
 
-    @Test
-    fun `migrate successful`(
-        @MockK sourceClient: BioWebClient,
-        @MockK targetClient: BioWebClient,
-        @MockK extSubmission: ExtSubmission
-    ) {
-        val request = MigrationRequest(
+    @Nested
+    @ExtendWith(MockKExtension::class)
+    inner class Migrate {
+
+        @MockK
```

Comment:
We should favor constructor parameters instead of lateinit variables. I think this change should be reverted

Reply:
is not possible to use mockK for constructor injection, it used to work before because it was a function parameter